tree/treeplayer/inc/TBranchProxyDirector.h

```diff
@@ -13,8 +13,10 @@
 #define ROOT_TBranchProxyDirector
 
 #include "Rtypes.h"
-
 #include <list>
+#include <vector>
```

Comment:
I do not see remaining uses of `<list>`. Please also remove the `R__OLDHPACC` fwd decl.